interface/themes/colors/style_ash_blue.scss

```diff
@@ -9,7 +9,7 @@ $pale: #B5BDDA;
 $midpalepale: #C3CAE1;
 $palepale: #E1E4F0;
 $palepurple: #E7E6F9;
-$paler: #F0F1F7;
```

Comment:
This is minor, but in the commit description I would include the fact that you updated a number of the colors for the themes just so people know better what this commit did.  The current description doesn't state that the themes were changed as part of this.